SwanCustomEnvironments/swancustomenvironments/scripts/builders/conda.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Create the environment and install packages
+${BUILDER_VERSION} create -p ${ENV_PATH} --file ${REQ_PATH} -y | tee -a ${LOG_PATH}
+
+# Activate the conda to recognize all conda commands in the shell
+ACTIVATE_CONDA_SHELL="source /opt/conda/etc/profile.d/conda.sh"
+$(echo ${ACTIVATE_CONDA_SHELL}) | tee -a ${LOG_PATH}
```

Comment:
What's the advantage of running these commands with variable definition + echo? It just adds more lines, no?

Reply:
We use the exact same command (for activating the environment) when we're creating it and each time the terminal is opened (we want to activate it as well)

Reply:
Yes I see the same pattern is used in multiple places, but does it help in any way? Wouldn't we save a few code lines just doing source | tee, with a comment before that says what the source does?

Reply:
I thought tee commands was only used for input and output purposes. How would you run a command and write itself on a file for running it again later?

Reply:
Doesn't
```
source /opt/conda/etc/profile.d/conda.sh | tee -a ${LOG_PATH}
```
achieve the same result as:
```
ACTIVATE_CONDA_SHELL="source /opt/conda/etc/profile.d/conda.sh"
$(echo ${ACTIVATE_CONDA_SHELL}) | tee -a ${LOG_PATH}
```
?

Reply:
Yup, but I was using it on makenv script too, that's why. But currently, we don't need to do that anymore